clients/web/src/views/widgets/FolderInfoWidget.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This view allows users to see and control access on a resource.
+ */
+girder.views.FolderInfoWidget = girder.View.extend({
+    initialize: function () {
+        this.needToFetch = !this.model.has('nItems') || !this.model.has('nFolders');
+        if (this.needToFetch) {
+            this.model.fetch({extraPath: 'details'}).once('g:fetched.details', function () {
```

Comment:
This branch isn't covered in the tests.  Is there a simple way to reach this code path?